### PR TITLE
adds polls and surveys content migration

### LINKED ISSUE #384: Polls and surveys content migration from v1 to v2
### For Polls
v1 <==> v2
polls_questions <==> questionnaires_poll
polls_choice <==> questionnaires_pollformfield

in v1 polls_questions has an attribute allow_multiple_choice which can be used to map to appropriate fields in v2
if allow_multiple_choice is true then field will be checkboxes
if allow_multiple_choice is false then there are two cases
  - poll question has 2 choices then field will be radio
  - poll question has more than 2 choices then field will be dropdown

### For surveys
v1 <==> v2
surveys_molosurveypage <==> questionnaires_surveys
surveys_molosurveyformfield <==> questionnaires_surveyformfield

in v1 and v2 skip logic and choices are handled differently but simple choices population could be done and actual skipping could be added manually.

### PR DESCRIPTION
**Pull request by sheralim012**
Fixes #384 

- migrates polls and surveys from v1 to v2
- adds index pages for polls, surveys, and quizzes

### CODE REVIEW

File: iogt_content_migration/management/commands/load_v1_db.py [hunk 1]
```diff
@@ -107,11 +117,13 @@ def migrate(self, root):
         self.migrate_images()
         self.load_page_translation_map()
         home = self.create_home_page(root)
-        section_index_page, banner_index_page, footer_index_page = self.create_index_pages(home)
+        section_index_page, banner_index_page, footer_index_page, poll_index_page, survey_index_page, quiz_index_page = self.create_index_pages(home)
```
Comment (ehmadzubair) [inline iogt_content_migration/management/commands/load_v1_db.py:120]: should we have the `create_index_pages` return a dictionary?

File: iogt_content_migration/management/commands/load_v1_db.py [hunk 2]
```diff
@@ -455,3 +476,205 @@ def stop_translations(self):
...
+
+        if choices_length == 2:
+            field_type = 'radio'
+        elif choices_length > 2:
+            if poll_row['allow_multiple_choice']:
+                field_type = 'checkboxes'
+            else:
+                field_type = 'dropdown'
+        else:
+            return
```
Comment (ehmadzubair) [inline iogt_content_migration/management/commands/load_v1_db.py:563]: logger?

File: iogt_content_migration/management/commands/load_v1_db.py [hunk 3]
```diff
@@ -455,3 +476,205 @@ def stop_translations(self):
...
+                    translated_survey.draft_title = row['draft_title']
+                    translated_survey.live = row['live']
+                    translated_survey.description = self.map_survey_description(row['description'])
+                    translated_survey.thank_you_text = self.map_survey_thank_you_text(row)
+                    translated_survey.allow_anonymous_submissions = row['allow_anonymous_submissions']
+                    translated_survey.allow_multiple_submissions = row['allow_multiple_submissions_per_user']
+                    translated_survey.submit_button_text = row['submit_text'] or 'Submit'
+                    translated_survey.direct_display = row['display_survey_directly']
+                    translated_survey.multi_step = row['multi_step']
+                    translated_survey.save(
```
Comment (ehmadzubair) [inline iogt_content_migration/management/commands/load_v1_db.py:605]: Maybe not add update fields? Somebody doing this later might miss it

File: iogt_content_migration/management/commands/load_v1_db.py (hunk 1 above)
Comment (sheralim012) [reply iogt_content_migration/management/commands/load_v1_db.py:120]: done

File: iogt_content_migration/management/commands/load_v1_db.py (hunk 2 above)
Comment (sheralim012) [reply iogt_content_migration/management/commands/load_v1_db.py:563]: added a message

File: iogt_content_migration/management/commands/load_v1_db.py (hunk 3 above)
Comment (sheralim012) [reply iogt_content_migration/management/commands/load_v1_db.py:605]: done

**Comment by sheralim012** [comment]
@geoo89 check and let me know if any change is required

**Pull request merged**